velox/exec/Operator.h

```diff
@@ -383,6 +383,15 @@ class Operator : public BaseRuntimeStatWriter {
   virtual void close() {
     input_ = nullptr;
     results_.clear();
+
+    // We are folding the operators background CPU time
+    // into OperatorStats.finishTiming.
+    const uint64_t backgroundCpuTimeMs = this->backgroundCpuTimeMs();
+    if (backgroundCpuTimeMs > 0) {
+      const CpuWallTiming opBackgroundTiming{1, 0, backgroundCpuTimeMs};
+      stats_.wlock()->finishTiming.add(opBackgroundTiming);
```

Comment:
Let's update OperatorStats::finishTiming to add comments to explain that it contains a sum of Operator::noMoreInput() and background CPU time.

CC: @xiaoxmeng

Reply:
I skipped this because we are no longer folding background CPU into finish time at the velox level.

Reply:
@patrickstuedi Per discussion with @mbasmanova offline, we could define a new field in OperatorStats say CpuWallTiming backgroundCpuTiming collected in Velox layer (velox/velox/exec/Operator.h), and convert (or merge into) the finishCpu in OperatorStats defined in presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h at Prestissimo layer.

Reply:
Sounds good, done.